zetaclient/chains/bitcoin/tx_script.go

```diff
@@ -192,6 +193,36 @@ func DecodeOpReturnMemo(scriptHex string, txid string) ([]byte, bool, error) {
 	return nil, false, nil
 }
 
+// DecodeScript decodes memo wrapped in an inscription like script in witness
```

Comment:
Not related to this PR directly but I would consider moving this entire file in an independent package: https://github.com/zeta-chain/node/issues/2525

But we can let as it in this PR to keep it minimal